src/OrderFormAcceptanceTests.Domain/Order.cs

```diff
@@ -174,14 +174,10 @@ public bool CanComplete()
                 && Progress.ServiceRecipientsViewed
                 && associatedServicesCount > 0;
 
-            var associatedServicesNoSolutions = catalogueSolutionsCount == 0
-                && associatedServicesCount > 0;
```

Comment:
I think we still have this in the API. Is it redundant now?

Reply:
It was one I added to force some tests to pass. Since modifying the persistence layer it's no longer relevant